scripts/ElaboratorListener_cpp.py

```diff
@@ -85,35 +85,20 @@ def _generate_module_listeners(models, classname):
           listeners.append(f'  inst->{method}(clone_vec);')
           listeners.append( '  for (auto obj : *vec) {')
           listeners.append( '    if (uniquifyTypespec()) {')
-          listeners.append( '      auto* stmt = obj->DeepClone(inst, context_);')
-          listeners.append( '      clone_vec->push_back(stmt);')
+          listeners.append( '      clone_vec->push_back(obj->DeepClone(inst, context_));')
           listeners.append( '    } else {')
           listeners.append( '      clone_vec->push_back(obj);')
           listeners.append( '    }')
           listeners.append( '  }')
           listeners.append( '}')
 
-        elif method in ['Ref_modules', 'Gen_stmts']:
-          pass # No elab
-
-        elif method not in ['Ports', 'Nets', 'Parameters', 'Param_assigns', 'Interface_arrays', 'Module_arrays']:
+        elif method not in ['Nets', 'Parameters', 'Param_assigns', 'Interface_arrays', 'Module_arrays']:
```

Comment:
Can we leave out this change?
It should be in a different PR, I'm not sure it is even correct.